pycytominer/tests/test_annotate.py

```diff
@@ -37,34 +37,63 @@ def test_annotate():
 
     # create expected result prior to annotate to distinguish modifications
     # performed by annotate to provided dataframes.
+    platemap_df = PLATEMAP_DF.copy()
```

Comment:
I'm a bit confused why this is necessary - our tests were passing before.

I think you can remove the copies and revert the changed lines back to the original data.

Reply:
In the live version of the code, tests are passing inappropriately because PLATEMAP_DF columns being renamed during the execution of `test_annotate()` is cancelled out by the inclusion of incorrect arguments in `test_annotate_output()` and `test_annotate_output_compress()`

### PLATEMAP_DF column names are modified during`test_annotate()`

Because `test_annotate()` uses the default argument `add_metadata_id_to_platemap=True`, the platemap column names are being modified during execution. This modification occurs [here](https://github.com/cytomining/pycytominer/blob/ea9873ddd3b7d9b7072cf7fc55d582012cf6d315/pycytominer/annotate.py#L78) in the load_platemap function. Calling `platemap_df = PLATEMAP_DF.copy()` and using the copied dataframe is necessary to prevent the column renaming from influencing other tests. 

### Incorrect Inputs
 Both `test_annotate_output()` and `test_annotate_output_compress()` contain the argument `add_metadata_id_to_platemap=False`. This means that for both of them, the lines `join_on=["Metadata_well_position", "Metadata_Well"]` should be changed to `join_on=["well_position", "Metadata_Well"]`  because "well_position" is the appropriate column name. Correcting these incorrect arguments will cause those tests to fail unless we modify `test_annotate()` with the line of code being discussed.. 





The reason that these tests were

Reply:
Amazing find @kenibrewer ! Thank you for bringing this to our attention.

I think this issue is worth resolving at the source. Can you add a data frame `.copy(deep=True)` to the `load_platemap()` function [here](https://github.com/cytomining/pycytominer/blob/ea9873ddd3b7d9b7072cf7fc55d582012cf6d315/pycytominer/cyto_utils/load.py#L78)?

Maybe something like:

```python
    if add_metadata_id:
        platemap_df = platemap.copy(deep=True)
        platemap_df.columns = [
            f"Metadata_{x}" if not x.startswith("Metadata_") else x
            for x in platemap_df.columns
        ]
    return platemap_df
```

(you'll need to modify `platemap_df = pd.read_csv(platemap, sep=delim)` a couple lines above as well.

I think that it's worth fixing this bug than fixing tests that work around the bug. (you'll also likely have to update tests that were inappropriately passing!)

Thanks again!

Reply:
Hi @gwaybio ,
I think you are correct about it being better to solve the problem by addressing the root cause of the bug which is the failure to use `.copy(deep=True)`.  I'll make the appropriate adjustments.